app/screens/localNode/LocalNode.js

```diff
@@ -1,28 +1,51 @@
 // @flow
 import React, { Component } from 'react';
-import { Route, Switch, Redirect } from 'react-router-dom';
 import styled from 'styled-components';
-import routes from '/routes';
+import { smColors } from '/vars';
 
 const Container = styled.div`
   width: 100%;
   height: 100%;
   padding: 60px 90px;
 `;
 
+const BodyWrapper = styled.div`
+  display: flex;
+  height: 100%;
+`;
+
+const Header = styled.span`
+  font-size: 31px;
```

Comment:
don't forget line-height param, in this case 42px

Reply:
ok